ceph_deploy/hosts/debian/__init__.py

```diff
@@ -18,7 +18,9 @@ def choose_init(module):
     """
     if distro.lower() == 'ubuntu':
         return 'upstart'
-    return 'sysvinit'
+    if not module.conn.remote_module.path_exists("/lib/systemd/system/ceph.target"):
+        return 'sysvinit'
+    return 'systemd'
```

Comment:
I know none of the LTS Ubuntu releases are on systemd yet, but since it is in 15.04, should we just switch this over to:

```
if ceph.target exists:
    systemd
elif ubuntu:
    upstart
else
    systemd
```

Thoughts?  That could be done later, of course.  What is here is fine.